packages/core/src/form-field/FormFieldLabel.css

```diff
@@ -12,6 +12,10 @@
   padding-top: var(--salt-spacing-100);
 }
 
+.saltFormField-labelRight .saltFormFieldLabel.saltText {
+  margin-left: auto;
```

Comment:
This type of change makes me worried the layout rules have changed and there will be a visual difference somewhere. I'll take a quick look to see if I can spot anything that isn't caught by our visual regression suite.